Formula/w/wownero.rb

```diff
@@ -35,7 +34,7 @@ class Wownero < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d08b8e26383ff1a7b709f65214b0f59505ca308f3b675bff55838c91de33a375"
   end
 
-  disable! date: "2025-05-11", because: "needs to use unmaintained `boost@1.85` and `protobuf@21`"
+  disable! date: "2025-05-11", because: "needs to use unmaintained `boost@1.85`"
```

Comment:
IMO this shouldn't be deprecated if `monero` isn't

Reply:
Given monero and wownero is not: let's keep this deprecated for the time being.